### DISCUSSION TITLE
option to create duplicate window / tab inside tabular editor window manager

### DISCUSSION THREAD

**Discussion by RHAWKEOMI** [TabularEditor/TabularEditor3#1216] [Ideas]
### Is your feature request related to a problem? Please describe

1) i want to be able to make a small change to a pivot grid without losing my current pivot grid. currently i have to create a new tab/window for the pivot grid and then rebuild the entire pivot grid manually which is very time consuming 
2) i want to add a new dax query with a few changes from my current dax query. i currently have to copy all, create new query, then paste all 

### Describe the solution you'd like

right clicking on a tab should offer a DUPLICATE option which creates a new window with the same content again in it

### Goals

1.
2.

**Comment by mlonsk** [comment]
Hi @RHAWKEOMI 

Thank you for the suggestion. I will discuss the feasibility with our development team, but I can not give a time line currently on any implementation.
